libpkg/pkgsign_ecc.c

```diff
@@ -813,6 +813,18 @@ _generate_private_key(struct ecc_sign_ctx *keyinfo)
 	return (EPKG_OK);
 }
 
+static inline void
+_specific_explicit_bzero(uint8_t *buf, size_t bufsz)
```

Comment:
why uint8_t and not void * both explicit_bzero and memset_s prototype are using uint8.

The CI is not happy about it on debian: https://builds.sr.ht/~bapt/job/1330026#task-build-658

Reply:
MB - fixed it now.